app/src/main/java/org/oppia/app/help/HelpActivity.kt

```diff
@@ -28,7 +28,7 @@ class HelpActivity : InjectableAppCompatActivity(), RouteToFAQListListener {
   companion object {
     // TODO(#1655): Re-restrict access to fields in tests post-Gradle.
     const val BOOL_IS_FROM_NAVIGATION_DRAWER_EXTRA_KEY =
-      "BOOL_IS_FROM_NAVIGATION_DRAWER_EXTRA_KEY"
```

Comment:
Rename this to 
```
const val IS_FROM_NAVIGATION_DRAWER_EXTRA_KEY = "HelpActivity.is_from_navigation_drawer"
```

else everything is correct.

Reply:
done